acl_upgrade.php

```diff
@@ -458,8 +458,11 @@
   addObjectAcl('patients', 'Patients', 'amendment' , 'Amendments (write,addonly optional)');
   // Add 'Lab Results (write,addonly optional)' object (added in 5.0.1)
   addObjectAcl('patients', 'Patients', 'lab'       , 'Lab Results (write,addonly optional)');
+  // Add 'Multipledb' object (added in 5.0.1)
+  addObjectAcl('admin', 'Administration', 'Multipledb', 'multipledb');
```

Comment:
Lets give this a human readable name :)
(and place in parenthesis which returns are supporting if pertinent; see other acos for examples)